Documentation/driver-api/virtual_bus.rst

```diff
@@ -0,0 +1,144 @@
+===============================
+Virtual Bus Devices and Drivers
+===============================
+
+See <linux/virtual_bus.h> for the models for virtbus_device and virtbus_driver.
+
+This bus is meant to be a minimalist software-based bus used for splitting the
+functionality of a PCI device into orthogonal parts. Each device attached
+to the bus represents an individual function of its parent PCI device. The
```

Comment:
Noooo.... 
This has nothing to do with PCI and should not be described as such.
Example: we can use the virtual bus with an ACPI-enumerated device, as we did for Baytrail IPC tests.

Reply:
@plbossart We're already getting a lot of thrash from the folks about using MFD and this will be only backfire. The reasoning would be if this is platform device, why not use the platform bus?

Reply:
because Greg KH said only use platform devices for DT/ACPI enumerated devices.

Reply:
@kv2019i @plbossart then should we clarify by something along the lines that the virtual bus is mainly to split up a single-function device into multiple child devices that do not exist physically and therefore cannot be DT/ACPI enumerated?